src/Signin.tsx

```diff
@@ -114,8 +115,12 @@ export class Signin extends PureComponent<Props, State> {
       clearInterval(this.intervalID)
 
       if (CLOUD) {
-        if (isFlagEnabled('useQuartzLogin')) {
-          window.location.reload()
+        if (
+          isFlagEnabled('useQuartzLogin') &&
+          process.env.NODE_ENV &&
+          process.env.NODE_ENV !== 'development'
```

Comment:
can this be shortened to

```ts
process.env?.NODE_ENV !== 'development'
``` 
?

Reply:
We also need to handle the case when the `NODE_ENV` is not set. This case is also considered a dev environment.